Bugzilla/Comment.pm

```diff
@@ -119,7 +119,7 @@ sub ES_SELECT_UPDATED_SQL {
 sub es_parent_id {
     my ($self) = @_;
 
-    return $self->bug_id,
+    return $self->ES_PARENT_CLASS->ES_TYPE . '_' . $self->bug_id,
```

Comment:
This code looks like a variation on a theme of `&es_id()` later in the patch. It might be worth a TODO comment about migrating it to use B:E:R:O:es_id() somehow instead of handcrafting, or not.